src/vm/i386/cgenx86.cpp

```diff
@@ -299,7 +299,12 @@ void TransitionFrame::UpdateRegDisplay(const PREGDISPLAY pRD)
 
     MethodDesc * pFunc = GetFunction();
     _ASSERTE(pFunc != NULL);
+
+#ifdef WIN64EXCEPTIONS
+    UpdateRegDisplayHelper(pRD, 0);
```

Comment:
@parjong I just want to confirm something. I think you've said before that the UnwindStackFrame didn't work if you have tried to call it with the esp set to the caller SP instead of the SP before the stack arguments were pushed. Am I right? And if yes, do you remember what was the issue?

I am asking since reading through the unwinder code, it seems to me that it could be possible to modify it (using ifdefs only for Linux) so that it would be able to unwind in the same way as the libunwind. That would remove the mismatch between these two unwinders, both would unwind to the caller SP.